A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/OPTIONS/A320_Neo_CDU_OPTIONS_Menu.js

```diff
@@ -22,8 +23,8 @@ class CDU_OPTIONS_MainMenu {
             ["<METAR SRC", `${storedAccelAlt} FT.>[color]cyan`],
             ["AOC[color]green", "DMC SELF-TEST[color]green"],
             ["<SIGMET SRC[color]inop", `${storedDMCTestTime} SEC.>[color]cyan`],
-            ["AOC[color]green"],
-            ["<TAF SRC"],
+            ["AOC[color]green", "DEFAULT BARO[color]green"],
+            ["<TAF SRC", `${storedDefaultBaroUnit}[color]cyan`],
```

Comment:
A suggestion for consistency with the other settings:
```suggestion
            ["<TAF SRC", `${storedDefaultBaroUnit}>[color]cyan`],
```